Adds new station: Lobster Station (Second PR, ignore the first one)

PubliclyExecutedPig opened the pull request:
Adds lobster station

<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
Lobster station, named because its shaped like a lobster, is a modern and expensive station, mainly focussed on research and development, income from passengers (like a resort), with a strong security to back it
## Why / Balance
To add some variation to the map pool, especially on apoapsis, whilst being quite unique in its characteristics... kinda like a dv fland station (but hopefully better, also this is a second PR bc idk how to update my first one's files)

## Technical details
<!-- Summary of code changes for easier review. -->

## Media
![Lobster-0](https://github.com/user-attachments/assets/5eeffe31-3cce-4960-aa26-d2adffa35dc7)


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).  
- [X] I have added media to this PR or it does not require an ingame showcase.

<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes

<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->:cl:
-add: Lobster Station, a medium pop station! (actual lobster is not guaranteed and is at the discretion of the chefs)

PubliclyExecutedPig committed:
Lobster Station

Adds lobster station

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

SolStar2 commented:
https://github.com/PubliclyExecutedPig/Delta-v/blob/430c1508e7f9c8541b98cd69d4ed1307ab739fe4/Content.IntegrationTests/Tests/PostMapInitTest.cs#L46-L64

You need to update the integration tests to include Lobster station. (edit: linked wrong code)

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
fixes stuff

hopefully this fixes the error thingy

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig committed:
Update PostMapInitTest.cs

PubliclyExecutedPig commented:
> https://github.com/PubliclyExecutedPig/Delta-v/blob/430c1508e7f9c8541b98cd69d4ed1307ab739fe4/Content.IntegrationTests/Tests/PostMapInitTest.cs#L46-L64
> 
> You need to update the integration tests to include Lobster station. (edit: linked wrong code)

well i've fixed that and the test still fails :/

PubliclyExecutedPig committed:
Update PostMapInitTest.cs

Signed-off-by: PubliclyExecutedPig <152017811+PubliclyExecutedPig@users.noreply.github.com>

PubliclyExecutedPig commented:
also btw the image is a bit outdated, i made tesla containment safer

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig committed:
Update PostMapInitTest.cs

Signed-off-by: PubliclyExecutedPig <152017811+PubliclyExecutedPig@users.noreply.github.com>

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
idk maybe fixes things

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Update lobster.yml

Signed-off-by: PubliclyExecutedPig <152017811+PubliclyExecutedPig@users.noreply.github.com>

PubliclyExecutedPig committed:
Update lobster.yml

maybe will fix it idk

Signed-off-by: PubliclyExecutedPig <152017811+PubliclyExecutedPig@users.noreply.github.com>

PubliclyExecutedPig committed:
Update lobster.yml

reverting previous fix that didnt fix 

Signed-off-by: PubliclyExecutedPig <152017811+PubliclyExecutedPig@users.noreply.github.com>

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig commented:
I HAVE NOW COMPLETELY 100% FINISHED LOBTER!!!!!

PubliclyExecutedPig committed:
THE REAL LAST COMMIT

ITS REAAAL

PubliclyExecutedPig committed:
last commit fr

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
lol

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
fix

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
last one i promise

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig commented:
fuck it. im fixing maints

PubliclyExecutedPig commented:
im hyping it too much im just adding some more rooms

PubliclyExecutedPig committed:
idk some more rooms plus librarian

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig commented:
ok like on my life im done updating lobter- its finished

PubliclyExecutedPig commented:
HOLY SHIT I FUCKED UP I ACCIDENTALLY REPLACED THE PROTOTYPE GIMME A SEC

PubliclyExecutedPig commented:
drafting till i fix

PubliclyExecutedPig committed:
fixed shit and added borgs

PubliclyExecutedPig committed:
OK FINAL COMMIT IM FR FR

ON MY LIFE LAST COMMIT

PubliclyExecutedPig commented:
fixed borgs - ALSO THATS MY LAST COMMIT LOBSTER IS FINISHED I PROMISE

Colin-Tel commented:
> fixed borgs - ALSO THATS MY LAST COMMIT LOBSTER IS FINISHED I PROMISE

mapping is never truly finished

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
lol- LAST ONE I PROMISE I JUST HAD TO FIX ONE DOOR WITH THE WRONG ACCESSES

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
deleted invalid stuff

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig committed:
botanist job spawn

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig commented:
i am finally ready

for real this time

PubliclyExecutedPig committed:
invalid fix

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig commented:
I changed my mind I will be doing some work on this map later

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig commented:
ive named cameras and apcs, and made some minor improvements

PubliclyExecutedPig commented:
so like remember when i was doing that whole thing about being finished but not really? im serious this time


unless velcroboy reviews it and finds something seriously wrong

:godo:

PubliclyExecutedPig commented:
![image](https://github.com/user-attachments/assets/5075b44a-b02b-4686-b95f-a3c6248c916a)

lobster origin story (emotional(

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
made lobster not 80 pop

PubliclyExecutedPig committed:
Merge branch 'map-testing' of https://github.com/PubliclyExecutedPig/Delta-v into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
fixed bridge wiring

after my most recent update i changed a location of an apc in the bridge but forgot to rewire the lv... silly 6,8 me

PubliclyExecutedPig committed:
fixed justice beacons

PubliclyExecutedPig committed:
added more rechargers to sec

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
removed all window lights

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
minor QoL

PubliclyExecutedPig committed:
Updates maints

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Added interrogation and reinforced some walls

PubliclyExecutedPig committed:
fixed new pipe colours and added some lights to maints rooms

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig commented:
once i work out these quirks im gonna be ready to merge

PubliclyExecutedPig committed:
maybe fixes stuff

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig commented:
idk why errors lol

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig commented:
YAY ALL ERRORS GONE

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
random super weapon

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
adds roboticist, portable scrubbers, and other minor QoL

PubliclyExecutedPig committed:
Update lobster.yml

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

deltanedas commented:
you dont need to merge master every 5 seconds :trollface:

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig commented:
> you dont need to merge master every 5 seconds :trollface:

you cant stop me

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
added lavaland

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
fixed arrivals

PubliclyExecutedPig committed:
Merge branch 'DeltaV-Station:master' into map-testing

PubliclyExecutedPig committed:
holopads + cargo assistant

PubliclyExecutedPig committed:
adds lobter to pools

deltanedas committed:
Merge branch 'master' into map-testing

Signed-off-by: deltanedas <39013340+deltanedas@users.noreply.github.com>

PubliclyExecutedPig committed:
Merge branch 'DeltaV-Station:master' into map-testing

PubliclyExecutedPig commented:
was vertex removed or something what

PubliclyExecutedPig committed:
fixed errors

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'DeltaV-Station:master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'DeltaV-Station:master' into map-testing

PubliclyExecutedPig committed:
stuff

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig committed:
Merge branch 'master' into map-testing

PubliclyExecutedPig commented:
i didnt mean to close it, whoopsies
